declarations/Twitter.history.json

```
@@ -0,0 +1,30 @@
+{
+    "Terms of Service": {
```

Each document type value is an array:
```suggestion
 "Terms of Service": [
    {
      "fetch": "https://twitter.com/tos",
      "remove": [],
      "select": [],
      "validUntil": ""
    }
  ],
  "Privacy Policy": [
    {
      "fetch": "https://twitter.com/en/privacy",
      "remove": [],
      "select": [],
      "validUntil": ""
    }
  ]
  ```